demoapp/src/main/java/me/uport/sdk/demoapp/MainListActivity.kt

```diff
@@ -0,0 +1,40 @@
+package me.uport.sdk.demoapp
+
+import android.content.Intent
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.widget.ArrayAdapter
+import android.widget.ListView
+
+/**
+ *
+ * Main screen for the demo app
+ * This lists all the features to be demonstrated within the app
+ * Clicking on an item opens up a new activity for the specific feature
+ *
+ **/
+
+class MainListActivity : AppCompatActivity() {
+
+    val features = arrayOf("Create an Account", "Manage Keys", "Create a JWT", "Verify a JWT", "Selective Disclosure")
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_list_main)
+
+        var listView = findViewById<ListView>(R.id.feature_list)
```

Comment:
generally `val` is preferred over `var`

In this case, the demoapp is already using the `kotlin-android-extensions` plugin which makes `findViewById` obsolete.
You can simply use something like:
 ```kotlin
import kotlinx.android.synthetic.main.activity_list_main.* //this import is added automagically by Android Studio

...
feature_list.adapter = adapter
feature_list.setOnItemClickListener {...}
```

Reply:
Nice! no more boiler plate